manifests/config/remove_default.pp

```diff
@@ -13,6 +13,13 @@
 # @example
 #   include holland::config::remove_default
 class holland::config::remove_default {
+  if $facts['os']['name'] == 'Ubuntu' {
```

Comment:
I don't have easy access to CentOS, RedHat, nor Scientific to test package installation, but I encountered problems with `holland backup` failing, as indicated by this manifest docstring until I removed the file.